src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -743,6 +743,27 @@ CHIP_ERROR DiscoveryImplPlatform::DiscoverCommissioners(DiscoveryFilter filter,
     return error;
 }
 
+CHIP_ERROR DiscoveryImplPlatform::StartDiscovery(DiscoveryType type, DiscoveryFilter filter, DiscoveryContext & context)
+{
+    CHIP_ERROR err = CHIP_ERROR_INVALID_ARGUMENT;
+
+    switch (type)
+    {
+    case DiscoveryType::kCommissionableNode:
+        err = DiscoverCommissionableNodes(filter, context);
```

Comment:
let's do `return DiscoverCommissionableNodes(filter, context)` and similar below. Make is clearer and uses up less lines:

```cpp
switch (type)
{
    case DiscoveryType::kCommissionableNode:
        return DiscoverCommissionableNodes(filter, context);
    case DiscoveryType::kCommissionerNode:
        return DiscoverCommissioners(filter, context);
    case DiscoveryType::kOperational:
        return CHIP_ERROR_NOT_IMPLEMENTED;
    default:
        return CHIP_ERROR_INVALID_ARGUMENT;
}
```

Reply:
updated